.drone.yml

```diff
@@ -19,38 +19,31 @@ pipeline:
       # Do not use quotes for variable value
       # because it does not work
       - DOCKER_BUILD_OPTS=--squash
-      - DOCKER_REPO=meteogroup/python3.7-geolambda
+      - DOCKER_REPO=meteogroup/geolambda-python37
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
     commands:
-        -  docker build -f Dockerfile.latest -t "$DOCKER_REPO:latest" $DOCKER_BUILD_OPTS .
-        -  docker build -f Dockerfile.1.0.0 -t "$DOCKER_REPO:1.0.0" $DOCKER_BUILD_OPTS .
+      - docker build -t "$DOCKER_REPO:${DRONE_TAG:-latest}" $DOCKER_BUILD_OPTS .
 
   run_tests:
-    image: meteogroup/python3.7-geolambda 
+    image: meteogroup/geolambda-python37 
     commands:
-      - gdalinfo ./tests/data/wavewatch_blacksea_6min_st2_gfs_fields_20190808_1200.nc
+      - python python/lambda/lambda_function.py 
 
-  publish_images:
+  docker:
     image: docker
     environment:
-      - IMAGE_LIST="meteogroup/python3.7-geolambda:latest meteogroup/python3.7-geolambda:1.0.0"
+      - DOCKER_REPO=meteogroup/geolambda-python37
     secrets:
       - DOCKER_USERNAME
       - DOCKER_PASSWORD
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
     commands:
       - docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
-      - |
-        for image_url in $IMAGE_LIST
-        do
-          echo -e "\n\n##########  Publish $image_url  ##########"
-          docker push "$image_url"
-        done
+      - docker push $DOCKER_REPO:${DRONE_TAG:-latest}
     when:
-      branch: master
-      event: push
+      event: tag
```

Comment:
Please also publish the image on a push to master. I am afraid you have to copy the whole step for this. Drone step conditions are not flexible enough to capture something like "tag or (push and branch==master)".
Maybe call this step `publish_tagged_image` and the new one `publish_latest_image`

Reply:
Good idea!